### PR TITLE
fix(session-manager): if connection is aborted leave session as is

### PR DESCRIPTION
**Pull request by cbernat**
### If the connection aborted by timeout leave session as it is
If the user is already logged in, and suddenly the connection is slower, it may happen that the connection is aborted due timeout. We had some entries like this, so we decided to keep the session as it is for this case.

### CODE REVIEW

File: src/services/session-manager.ts [hunk 1]
```diff
@@ -103,7 +112,6 @@ export class OnlineSessionManager implements SessionManager {
           return;
         }
       }
-      this.updateSession({ status: 'non-authenticated' });
```
Comment (andresmoschini) [inline src/services/session-manager.ts:106]: What happens if the timeout occurs in the first request (when the status is _unknown_ and a spinner is being shown)?

Will be the spinner visible up to the next call to _GetUserData_?

Comment (cbernat) [reply src/services/session-manager.ts:106]: When status is unknown, account is empty, there is no user logged, so session value would be 'non-authenticated'
We only do nothing if the connection timed out and a user is logged in. 
In the first case, status goes from 'unknown' to 'non-authenticated' when timeout fails.
image

I don´t see a way that can happen that user account is set, and it's the first time it's called. First time account shouldn´t be logged.

Comment (andresmoschini) [reply src/services/session-manager.ts:106]: Great explanation! I understand now, thanks!

**Pull request merged**